.github/workflows/coding-standards.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           restore-keys: "php-${{ matrix.php-version }}-composer-locked-"
 
       - name: "Install dependencies with Composer"
-        run: "composer install --no-interaction --no-progress --no-suggest"
+        run: "composer update --no-interaction --no-progress --prefer-dist --prefer-stable"
```

Comment:
This file seems quite a bit out of sync with [upstream](https://github.com/doctrine/.github/blob/main/workflow-templates/coding-standards.yml), let's use the same solution instead?

Reply:
Now I remembered that I tried, getting this:

https://github.com/doctrine/DoctrineMongoDBBundle/runs/1787972990?check_suite_focus=true

As the minimum stability is `dev`, it installs [`dev-master` of `slevomat/coding-standard`](https://github.com/doctrine/DoctrineMongoDBBundle/runs/1787972990?check_suite_focus=true#step:4:69) which removes [`SlevomatCodingStandard.Classes.UnusedPrivateElements`](https://github.com/slevomat/coding-standard/commit/655524a1a70b6b1cc0f75bb5c1b7c6b8286dad86).

The `dev` stability was added in https://github.com/doctrine/DoctrineMongoDBBundle/pull/602 to apparently test dev versions in travis and it was not added in #660, so maybe a solution could be to remove this `dev` stability.

Reply:
Yes, adding a step that does `composer config minimum-stability stable` could be a solution.